### ISSUE TITLE
Make File Icon Per File Types Optional

### ISSUE THREAD

**Issue by Astrantia** [Microsoft/vscode#58913]
I really like them, but I already see part of the people complaining about it and as I don't want vscode to revert the commit that introduced them, can we make them optional during setup?

@bpasero

**Comment by Astrantia** [comment]
@bpasero I should've written "I *can* already see part of the people complaining about it". I'm sure some people will complain as it's more than just a 'small' update

**Comment by bpasero** [comment]
@Astrantia yeah I get that, but I was wondering why people would want an option. E.g. what could be a reason for not wanting the icons?

**Comment by Astrantia** [comment]
@bpasero The provided icons are not customizable and some people may like the ones provided by you and some may not. Most people are used to having no icons at all and a change that updates them without their permission is going to make some unhappy.

**Comment by bpasero** [comment]
Actually I would assume that this would work: 

![image](https://user-images.githubusercontent.com/900690/45760078-90927b00-bc29-11e8-9f5b-67b77dd14762.png)

@joaomoreno to confirm

**Comment by joaomoreno** [comment]
That will skip associating files with Code. I think the request is to keep the association but not have custom icons per file type.
